extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -267,6 +267,18 @@ export class AmpA4A extends AMP.BaseElement {
     this.creativeBody_ = null;
 
     /**
+     * Initialize this with the slot width/height attributes, and override
+     * later with what the network implementation returns via
+     * extractCreativeAndSignature.
+     *
+     * @private {?{width, height}}
+     */
+    this.creativeSize_ = {
+      width: Number(this.element.getAttribute('width')),
```

Comment:
So what happens if attribute value is 'auto'?  This would be isNaN, right?  We shouldn't really on the network overriding size via extractCreativeAndSignature (for all we know we'll break Cloudflare!)